### ISSUE TITLE
Back-propagation

### ISSUE THREAD

**Issue by CohenQU** [CohenQU/Artificial-Intelligence-Guide#14]
### DeepLizard
- [Backpropagation explained | Part 1 - The intuition](http://deeplizard.com/learn/video/XE3krf3CQls)
- [Backpropagation explained | Part 2 - The mathematical notation](http://deeplizard.com/learn/video/2mSysRx-1c0)
- [Backpropagation explained | Part 3 - Mathematical observations](http://deeplizard.com/learn/video/G5b4jRBKNxw)
- [Backpropagation explained | Part 4 - Calculating the gradient](http://deeplizard.com/learn/video/Zr5viAZGndE)
- [Backpropagation explained | Part 5 - What puts the "back" in backprop?](http://deeplizard.com/learn/video/xClK__CqZnQ)

### MIT Opencourseware
- [Lecture 12A: Neural Nets](https://www.youtube.com/watch?time_continue=1632&v=uXt8qF2Zzfo#t=27m00s) (From 27:00)
